be/src/exec/connector_scan_node.cpp

```diff
@@ -193,8 +130,10 @@ pipeline::OpFactories ConnectorScanNode::decompose_to_pipeline(pipeline::Pipelin
                                 context->next_operator_id(), this, runtime_state(), dop, std::move(buffer_limiter),
                                 is_stream_pipeline);
 
-    scan_op->set_estimated_mem_usage_per_chunk_source(_estimated_mem_usage_per_chunk_source);
-    scan_op->set_scan_mem_limit(_mem_limit);
+    scan_op->set_chunk_source_mem_bytes(_estimated_chunk_source_mem_bytes +
+                                        _estimated_scan_row_bytes * runtime_state()->chunk_size());
```

Comment:
because chunk itself also consumes some memory.

so memory cosumption of a chunk source includes:
- chunk source interal memory usage
- chunk that it outputs